modules/info.py

```diff
@@ -0,0 +1,100 @@
+"""MIT License
+
+Copyright (c) 2021-Present PythonistaGuild
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+"""
+from __future__ import annotations
+
+import discord
+from discord.ext import commands
+import core, typing
+#from typing import TYPE_CHECKING, Any
+
+class InformationEmbed(discord.Embed):
+    """A subclass of discord.Embed. 
+    
+    This class allows to automatically get information within the class instead of recreating the embed each time
+    
+    :param author: The embed author. Expects discord.Member or discord.User
+    :param entity: The Member, User, Role, TextChannel, or Guild to get information about.
+    """
+    def __init__(self, author: typing.Union[discord.Member, discord.User], entity: typing.Union[discord.Member, discord.User, discord.Role, discord.TextChannel, discord.Guild]):
```

Comment:
Make these kwargs as there is a large number of parameters.

Reply:
If I make these kwargs how can I still ensure that the values are valid Union types?